src/lsdb/core/crossmatch/kdtree_match.py

```diff
@@ -18,7 +18,7 @@ class KdTreeCrossmatch(AbstractCrossmatchAlgorithm):
     def crossmatch(
         self,
         n_neighbors: int = 1,
-        d_thresh: float = 0.01,
+        d_thresh: float = 0.000278,
```

Comment:
Can we add a comment saying this represents 1 arc second, to make it more explicit?